src/universal/components/AnimatedFade.js

```diff
@@ -0,0 +1,63 @@
+import {css} from 'aphrodite-local-styles/no-important';
+import React, {Component} from 'react';
+import CSSTransition from 'react-transition-group/CSSTransition';
+import withStyles from 'universal/styles/withStyles';
+
+class AnimatedFade extends Component {
```

Comment:
facebook's CSSTransition V2 is actually pretty great. it doesn't do any animating, it just provides the hooks. then, you can act on those hooks with css, a tweening lib, etc.